src/modules/contentful/touts/contentfulTout.base.js

```diff
@@ -13,22 +13,48 @@ class BaseContentfulTout extends Component {
         description,
         heroButtons,
         media
-      }
+      },
+      displayTitle,
+      searchTerm,
+      productsFound
     } = this.props
-    return (
-      <ContentfulRenderer {...media}>
-        <div className='roa-tout-overlay'>
-          <MirageMarkdown>{description}</MirageMarkdown>
-          <div className='roa-tout-buttons'>
-            {heroButtons && heroButtons.map((button) => {
-              return(
-                <ContentfulRenderer {...button} key={button.sys.id} />
-              )
-            })}
+
+    // If searchTerm or displayTitle present, do not render hero image
+    if (searchTerm) {
+      let searchTitle = `You searched ‘${searchTerm}’`
+      if (!productsFound) searchTitle = `Aw, shucks! 0 results for your search ‘${searchTerm}.’`
```

Comment:
not sure how i feel about these one liners as opposed to ` if () { ... }` multiline brackets we use everywhere else but ill leave that open to discussion

Reply:
Good to keep it consistent with rest of codebase, we can even customize a linter to catch situations like this